pdp/static/js/__test__/app-test-helpers.js

```diff
@@ -0,0 +1,18 @@
+function initializeGlobals() {
+    window.pdp = {
+        VERSION: '0.1',
+        app_root: 'https://data.pacificclimate.org/portal',
+        data_root: 'https://data.pacificclimate.org/data',
```

Comment:
So does this mean tests will be run using the live backend, and not the development backend? I don't think that's a huge issue that needs to be corrected now, but maybe make a note of that fact so that a hypothetical person changing the back end and front end in tandem would understand why they're getting anomalous test results?

Reply:
Augh, you're right! Tests should not depend on live data, for the reason you give! More mocking! Augh!!

Reply:
Actually, I was right. These just match the mock data, which looks like but is not real data. Yay. Fixing to make more apparent.